packages/perseus/src/widgets/orderer/orderer-validator.test.ts

```diff
@@ -0,0 +1,54 @@
+import {act} from "@testing-library/react";
+
+import {renderQuestion} from "../__testutils__/renderQuestion";
```

Comment:
Ideally we would keep React-centric tests where they are and have non-React tests specifically for the validator.

Reply:
Will fix this for sure.